polar-core/src/data_filtering.rs

```diff
@@ -316,6 +318,30 @@ impl VarInfo {
     }
 }
 
+fn data_filtering_error<A>(msg: String) -> PolarResult<A> {
+    Err(RuntimeError::DataFilteringError(msg).into())
+}
+
+fn unregistered_field_error<A>(var_type: &str, field: &str) -> PolarResult<A> {
+    let msg = formatdoc!(
+        r#"Unregistered field or relation: {}.{}
+        
+        Please include `{}` as a field in your `register_class` call:
+
+        oso.register_class({}, fields={{
```

Comment:
Love the error message, but this could be a bit confusing for non-Python people.
Maybe we could say, "e.g. in Python" and link to docs?

Reply:
Alternatively, this would be one reason to actually handle the error variant differently.

Reply:
Good point. I think adding "e.g. in Python" & a docs link would be ok for now, but esp. as we add more long explicit errors it'd make sense to have a standard way for the host to do its own formatting etc.

Reply:
@gj -- potentially relevant to your interests

Reply:
oof, another angle to think about